.travis.yml

```diff
@@ -1,13 +1,11 @@
 language: ruby
 
 rvm:
-  - 2.1
-  - 2.2
   - 2.3
   - 2.4
   - 2.5
   - 2.6
-  - jruby-9.0.5.0
+  - jruby-9.2.7.0
```

Comment:
Had to bump this to get the build passing.

According to [their wiki](https://github.com/jruby/jruby/wiki/Roadmap), 9.0 is Ruby 2.2 compatible, 9.1 is Ruby 2.3 compatible, and 9.2 is Ruby 2.5 compatible.

I think that means that we could just go to JRuby 9.1 and get this passing, but I opted to jump to the latest given that it looks like 9.1 may not technically be supported anymore anyway.